### PR TITLE
Error in AlignedImageWidget

### LINKED ISSUE #737: Error Dialog in AlignedImageWidget incorrectly displayed
#### Problem description
In an AlignedImageWidget , we get an error for no reason due to a bug in the code 

#### Steps to reproduce the problem
Just using AlignedImageWidget in the form
issue_odk

#### Other information 
This is due to a simple bug in the AlignedImageWidget code 

And I am working on it !

### PR DESCRIPTION
**Pull request by rishsethia**
This PR fixes the bug in AlignedImageWidget

### CODE REVIEW

File: collect_app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -124,6 +124,7 @@
         <intent-filter>
             <action android:name="android.intent.action.VIEW" />
             <action android:name="android.intent.action.EDIT" />
+            <action android:name="android.intent.action.PICK" />
```
Comment (lognaturel) [inline collect_app/src/main/AndroidManifest.xml]: This is an unrelated change, please remove it from this pull request! 

You should create branches for each pull request as described in the contribution guide so that you can work on several different things at the same time without introducing this kind of change.

Comment (rishsethia) [reply collect_app/src/main/AndroidManifest.xml]: Hello @lognaturel ,

Yes mam , Got that ! 
Thanks

**Comment by rishsethia** [comment]
@lognaturel what's the status of this pr ?

**Comment by lognaturel** [comment]
@rishsethia Please be sure to join us in [Slack](http://slack.opendatakit.org) to stay up to date with the latest! We are currently in feature freeze until Sunday. After the Sunday release, we will resume code review of open pull requests that aren't bug fixes.

**Comment by lognaturel** [comment]
Sorry, @rishsethia, I misunderstood what this was. I have a couple of questions that I'll take to the issue.

**Comment by lognaturel** [comment]
I'll merge this now since it does seem like @rishsethia's fix better captures the intent of the splits and I'll file a separate issue for deprecating the widget in the next release.
